SDK/MobileCenter/Microsoft.Azure.Mobile.Android/Resources/Resource.Designer.cs

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Fixed - problem was that in .gitignore we ignore Resource.designer.cs, not Resource.Designer.cs